src/Fynance.Tests/TickerTests.cs

```diff
@@ -134,7 +134,7 @@ public void Should_define_an_interval_of_dates_by_set_interval()
 			var startDate = new DateTime(2020, 1, 1);
 			var finishDate = new DateTime(2020, 5, 31);
 
-			var result = Ticker.Build().SetInterval(startDate, finishDate);
+			var result = Ticker.Build().SetPeriod(startDate, finishDate);
```

Comment:
There is a huge different between interval and period. The interval is a range between of dates you want to define to get data from the data source, from a start date and finish date. The Period is the slices of time between each entry, for example: Each entry should represent `OneHour` or `OneDay` etc. Your PR is great but undo this method name change and let's keep the `1.0.5` version. I will run some tests against it latter and publish this new version into nuget. :)

Reply:
reverted it.
But also I think it's wrongly named https://github.com/felipeoriani/Fynance/issues/4
I understand the difference between Period and Interval and in that very specific case SetInterval is only changing the period 
![image](https://user-images.githubusercontent.com/35202750/134673961-168d775d-41cd-41d5-80af-f0c076aa9549.png)

Reply:
I will check it later and after publish on nuget a new version.
Thanks again for your contributions Erwan, I really appreciate them.

Reply:
There might be a bug here @felipeoriani as based on the code this is the oposite
```      
 .SetInterval(Fynance.Interval.OneDay)
.SetPeriod(Fynance.Period.TenYears);
```
this will return 3650 results so Interval is slice of time and Period is how long to get the results for